examples/all_in_one/slo-exporter/conf/slo_rules.yaml

```
@@ -0,0 +1,39 @@
+rules:
+  - slo_matcher:
+      domain: example-domain
+    failure_conditions:
+      - operator: numberIsEqualOrHigherThan
+        key: statusCode
+        value: 500
+    additional_metadata:
+      slo_type: availability
+      slo_version: 6
```

```suggestion
      slo_version: 1
```
Why to start from version=6? Internally I know why you/we use it currently:) But it's not relevant in public repo.

---

Fixed within https://github.com/seznam/slo-exporter/pull/4#discussion_r458088070